CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ and this project adheres to
 - cosmwasm-std: Implement `std::ops::Sub` for `math::Decimal`
 - cosmwasm-std: Add `Timestamp::seconds` and `Timestamp::subsec_nanos`.
 - cosmwasm-std: Implement division for `Decimal / Uint128`
+- cosmwasm-std: Fix `Uint64::multiply_ratio` and `Uint128::multiply_ratio`
+  so that internal multiplication cannot cause an unnecessary overflow. ([#920])
```

Comment:
`[#920]` is a link notation for Markdown. But you did not add the link target. You need to add:

```
[#920]: https://github.com/CosmWasm/cosmwasm/issues/920
```

Reply:
Oh yeah, should this be under `### Changed`, too?

Reply:
Yes. Ideally under `### Fixed`, see https://keepachangelog.com/en/1.0.0/